src/components/converter/Converter.tsx

```diff
@@ -4,22 +4,22 @@ import checkBinary from "../../helpers/checkBinary";
 import styles from "./converter.module.css";
 
 function Converter() {
-  const [userInput, setUserInput] = useState("");
+  const [output, setOutput] = useState("");
 
   function handleChange(e: React.ChangeEvent<HTMLInputElement>) {
     const userInputValue = checkBinary(e.target.value);
     if (userInputValue) {
```

Comment:
Same as the comment below, you don't really need to store this result as you only use it once.

```
if (checkBinary(e.target.value))
```